download.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: download
+title: "Download - Jamulus"
+lang: "en"
+permalink: /download
+
+---
+
+
```

Comment:
Any reasons for this spacing here?

Also: would it be possible to use markdown here?

Reply:
You mean for the entire page instead of HTML?

Reply:
> You mean for the entire page instead of HTML?

Use as much Markdown as possible since it will make reading much easier for e.g. translators.

You can add html to .md files. See: https://stackoverflow.com/a/35614879

Reply:
For a mostly text-based page I think .md would be fine, but I found the HTML formatting confusing when there's a lot on the page.

Reply:
Fair point.